backend/routes/officer_agency_search.py

```diff
@@ -0,0 +1,108 @@
+from sqlalchemy.sql.functions import GenericFunction
+
+from backend.database.models.user import UserRole
+from ..database import SearchView, db
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended.view_decorators import jwt_required
+from backend.auth.jwt import min_role_required
+
+
+class TSRank(GenericFunction):
+    package = 'full_text'
+    name = 'ts_rank'
+    inherit_cache = True
+
+
+bp = Blueprint("search_route", __name__, url_prefix="/api/v1/search")
+"""
+TODO:
+Convert search function to endpoint,
```

Comment:
Is there another endpoint you're planning or is this TODO satisfied with this PR?

Reply:
Yes, the TODO is satisfied. Forgot to remove it.